spring-boot/src/main/java/io/vanillabp/camunda8/wiring/Camunda8TaskWiring.java

```diff
@@ -113,11 +113,10 @@ public void openWorkers() {
                             .jobType("io.camunda.zeebe:userTask")
                             .handler(userTaskHandler)
                             .timeout(Integer.MAX_VALUE) // user-tasks are not fetched more than once
-                            .name(workerId)
-                            .tenantId(tenantId);
+                            .name(workerId);
                     final var workerProperties = camunda8Properties.getUserTaskWorkerProperties(workflowModuleId);
                     workerProperties.applyToUserTaskWorker(userTaskWorker);
-                    return userTaskWorker;
+                    return tenantId != null ? userTaskWorker.tenantId(tenantId) : userTaskWorker;
```

Comment:
Please change this line into a separate if-statement: Using a tenaer operate is OK in general but not here in combination with the return statement. It tells that different responses are expected depending on the given condition. But actually, the worker needs to be configured properly depending on the given condition regardless whether it is return by the method or not. Additionally, I would place it underneath line 116 since the tenant is an crucial configuration as the other aspects above line 116 but the configuration in 117/118 are nice-to-haves:

```java
        ...
        .name(workerId);

    if (tenantId != null) {
        userTaskWorker.tenantId(tenantId);
    }

    final var workerProperties = ....
```